js2-mode.el

```diff
@@ -6163,8 +6163,6 @@ its relevant fields and puts it into `js2-ti-tokens'."
                              (setf (js2-token-beg token) (- js2-ts-cursor 2))
                              (js2-skip-line)
                              (setf (js2-token-comment-type token) 'line)
-                             ;; include newline so highlighting goes to end of window
```

Comment:
> `//` comments stretch until the end of the line

That does make sense.

Okay, I will try special-casing for comment nodes elsewhere.